code/game/objects/structures/morgue.dm

```diff
@@ -14,18 +14,18 @@
 	name = "morgue"
 	desc = "Used to keep bodies in until someone fetches them."
 	icon = 'icons/obj/stationobjs.dmi'
-	icon_state = "morgue1"
+	icon_state = "morgue"
 	density = 1
 	max_integrity = 400
 	dir = EAST
 	var/obj/structure/m_tray/connected = null
 	var/list/status_descriptors = list(
-	"The tray is currently extended.",
-	"The tray is currently empty.",
-	"The tray contains an unviable body.",
-	"The tray contains a body that is responsive to revival techniques.",
-	"The tray contains something that is not a body.",
-	"The tray contains a body that might be responsive."
+	"extended" = "The tray is currently extended.",
```

Comment:
Could you make defines out of these?

Reply:
I couldn't really figure out how to do it while making it work with the overlay names at the same time. If they were only used for descriptions, then yes defines would have been better.